### PR TITLE
add initial test automation and health check

### PR DESCRIPTION
**Pull request by mistergreen22**

### CODE REVIEW

File: tests/functional/tests/constants.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+MODULE_NAME = 'AlienVault OTX'
+OBS_HUMAN_READABLE = {
```
Comment (gdavoian) [inline tests/functional/tests/constants.py:2]: What is `obs`? Observation, obsession, obsidian?

Comment (mistergreen22) [reply tests/functional/tests/constants.py:2]: observer ward)

File: tests/functional/tests/test_refer.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from urllib.parse import quote
+from ctrlibrary.core.utils import get_observables
+from ctrlibrary.threatresponse.enrich import enrich_refer_observables
+from tests.functional.tests.constants import (
+    MODULE_NAME,
+    OBSERVABLE_HUMAN_READABLE_NAME
+)
+
+
+@pytest.mark.parametrize(
```
Comment (oshtaier) [inline tests/functional/tests/test_refer.py:11]: Just an idea:
```
@pytest.mark.parametrize(
    'human_readable_observable_type, observable',
    (('IP', '54.38.157.11'),
     ('IPv6', '2620:12f:c000:0:92e2:baff:fecd:3f94'),
     ('URL', 'http://blockchains.pk/nw_NIHbAj35.bin'),
     ('email', 'ysadmin@meraki.com'),
     ('domain', 'jsebnawkndwandawd.sh'),
     ('SHA256',
      'af689a29dab28eedb5b2ee5bf0b94be2112d0881fad815fa082dc3b9d224fce0'),
     ('MD5', 'f8290f2d593a05ea811edbd3bff6eacc'),
     ('SHA1', 'da892cf09cf37a5f3aebed596652d209193c47eb'))
)
def test_positive_smoke_enrich_refer_observables(module_headers, observable,
                                                 observable_type):
    """Perform testing for enrich refer observable endpoint to check status of
    AlienVault OTX module
    ID: CCTRI-1336-1f700099-447c-4803-9df7-d1c97cc5abdb
    Steps:
        1. Send request to enrich refer observable endpoint
    Expectedresults:
        1. Response body contains refer entity with needed fields from
        AlienVault OTX module
    Importance: Critical
    """
    observable_type = lower(human_readable_observable_type)
    observables = [{'type': observable_type, 'value': observable}]
    response_from_all_modules = enrich_refer_observables(
        payload=observables,
        **{'headers': module_headers}
    )['data']
    response_from_alien_vault = get_observables(response_from_all_modules,
                                                MODULE_NAME)
    assert response_from_alien_vault['module'] == MODULE_NAME
    assert response_from_alien_vault['module_instance_id']
    assert response_from_alien_vault['module_type_id']
    assert response_from_alien_vault['id'] == (
        f'ref-avotx-search-{observable_type}-{quote(observable, safe="")}')
    assert response_from_alien_vault['title'] == (
        f'Search for this {human_readable_observable_type}')
    assert (response_from_alien_vault['description']) == (
        f'Lookup this {human_readable_observable_type} on '
        f'{MODULE_NAME}')
    assert response_from_alien_vault['categories'] == [MODULE_NAME, 'Search']
```

so in this way, you would not hardcode values in separate list of constants, so you can expand your list of data without changing other modules

Comment (mistergreen22) [reply tests/functional/tests/test_refer.py:11]: I thought about something like this, but if human readable type would be for example "IP address" we couldn't use just lover case. So I prefer current variant.

**Pull request merged**